src/pages/About/About.page.tsx

```diff
@@ -1,5 +1,23 @@
+import React from "react";
+import FeaturesCards from "./components/FeaturesCards/FeaturesCards";
+import { Typography } from "@/components/UI/Typography/Typography";
+import { Container } from "@mantine/core";
+
 const AboutPage = () => {
-  return <div>AboutPage</div>;
+  return (
+    <Container size="lg" py="xl">
+      <section style={{ textAlign: "center", marginBottom: "2rem" }}>
```

Comment:
This entire code block should be a component, with the constants coming from the constants folder. 
Also avoid inline style and mantine ui props like `size="lg"`, why do you need this container?